io.github.giantpinkrobots.bootqt.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.giantpinkrobots.bootqt
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+
+command: bootqt.sh
+
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
I think using `flatpak-spawn` is questionable approach given projects like https://flathub.org/apps/details/org.fedoraproject.MediaWriter manage just fine without it.

Reply:
Is it bad to use flatpak-spawn? I just use the dd command to write the image file to the USB drive. Does it create a security issue?

Reply:
It is considered wrong approach as it completely throws out any sandbox guarantees. I mention MediaWriter as it does the same thing in a fully sandboxed way.

Reply:
I understand. So everything on Flathub is fully sandboxed? I don't know how Fedora Media Writer does it. If it's not possible to release non-sandboxed software on Flathub I'll search around and try to make it run in a sandbox.